tortoise/contrib/test/__init__.py

```diff
@@ -238,3 +239,29 @@ class TestCase(SimpleTestCase):
     async def _tearDownDB(self) -> None:
         _restore_default()
         await self.transaction.rollback()
+
+
+def requireCapability(**conditions):
```

Comment:
Curious question: why the `camelCase` here (and in other functions in this module)?

Reply:
the unittest module is full of camelcase. Some examples:
* setUp
* skipIf
* skipUnless
* tearDown
* expectedFailure